README.md

```diff
@@ -21,7 +21,6 @@ Algo VPN is a set of Ansible scripts that simplify the setup of a personal Wireg
 * Does not support legacy cipher suites or protocols like L2TP, IKEv1, or RSA
 * Does not install Tor, OpenVPN, or other risky servers
 * Does not depend on the security of [TLS](https://tools.ietf.org/html/rfc7457)
-* Does not require client software on most platforms
```

Comment:
This is still true for IKEv2 and Apple devices, maybe we need to rephrase this just a bit

Reply:
You should specify it's only for Apple clients and NOT available for Windows

Reply:
Ok, hope this is better. Also I fixed the spelling of [IPsec](https://blogs.technet.microsoft.com/ianhamer/2006/02/10/an-ipsec-is-an-ipsec-is-an-ipsec/) and WireGuard.